### ISSUE TITLE
Extract:true gives wrong output on absolute paths, and if an external stylesheet is in the markup

### ISSUE THREAD

**Issue by alanontheweb** [addyosmani/critical#61]
NOTE - This was only tested with the new Stream functionality.

Example - Using the new Stream functionality and Extract:true, Gulp is globbing through several HTML files, and some are in subdirectories. In my HTML, my stylesheet path is "/assets/css/app.min.css" (note the slash at the beginning).

This does not work or result in generating multiple partial CSS files, and it doesn't append the appropriate filename to the LoadCSS snippet functions.

Removing the front slash to just "assets/css/app.min.css" MAKES THE WHOLE THING WORK. However, once the files are outputted, the loadCSS is pointing to a relative asset, and this breaks the pages that are inside a subdirectory, since it can't find the CSS there.

to recap - It does work with both absolute and relative paths, as long as you don't use the extract function (which creates its own unique stylesheet per page). If you use extract: true, then it only works with a relative path.

Also, with extract: true, the loadCSS function appends no path for the stylesheet.

Lastly, extract:true also gets hung up if an external stylesheet is defined in the markup.

**Comment by bezoerb** [comment]
@alanontheweb just added a fix to `inline-critical`. Want to give it another try?

**Comment by bezoerb** [comment]
@alanontheweb any updates here?

**Comment by bezoerb** [comment]
Closing this due to lack of feedback. Feel free to reopen if the problem still exists
